applications/CompressiblePotentialFlowApplication/custom_response_functions/adjoint_lift_response_function_coordinates_jump.cpp

```diff
@@ -0,0 +1,164 @@
+                                   const Matrix& rResidualGradient,
+                                   Vector& rResponseGradient,
+                                   const ProcessInfo& rProcessInfo)
+    {
+        KRATOS_TRY;
+        rResponseGradient = ZeroVector(rResidualGradient.size1());
+        KRATOS_CATCH("");
+    }
+  
+
+    void AdjointLiftJumpCoordinatesResponseFunction::CalculatePartialSensitivity(Element& rAdjointElement,
+                                             const Variable<double>& rVariable,
+                                             const Matrix& rSensitivityMatrix,
+                                             Vector& rSensitivityGradient,
+                                             const ProcessInfo& rProcessInfo)
+    {
+        KRATOS_TRY;
+        rSensitivityGradient = ZeroVector(rSensitivityMatrix.size1());
+        KRATOS_CATCH("");
+    }
+
+    void AdjointLiftJumpCoordinatesResponseFunction::CalculatePartialSensitivity(Condition& rAdjointCondition,
+                                             const Variable<double>& rVariable,
+                                             const Matrix& rSensitivityMatrix,
+                                             Vector& rSensitivityGradient,
+                                             const ProcessInfo& rProcessInfo)
+    {
+        KRATOS_TRY;
+        rSensitivityGradient = ZeroVector(rSensitivityMatrix.size1());
+        KRATOS_CATCH("");
+    }
+
+    void AdjointLiftJumpCoordinatesResponseFunction::CalculatePartialSensitivity(Element& rAdjointElement,
+                                             const Variable<array_1d<double, 3>>& rVariable,
+                                             const Matrix& rSensitivityMatrix,
+                                             Vector& rSensitivityGradient,
+                                             const ProcessInfo& rProcessInfo)
+    {
+        KRATOS_TRY;
+        rSensitivityGradient = ZeroVector(rSensitivityMatrix.size1());
+        KRATOS_CATCH("");
+    }
+
+    void AdjointLiftJumpCoordinatesResponseFunction::CalculatePartialSensitivity(Condition& rAdjointCondition,
+                                             const Variable<array_1d<double, 3>>& rVariable,
+                                             const Matrix& rSensitivityMatrix,
+                                             Vector& rSensitivityGradient,
+                                             const ProcessInfo& rProcessInfo)
+    {
+        KRATOS_TRY;
+        rSensitivityGradient = ZeroVector(rSensitivityMatrix.size1());
+        KRATOS_CATCH("");
+    }
+
+    void AdjointLiftJumpCoordinatesResponseFunction::GetNeighboringElementPointer()
+    {
+        KRATOS_TRY;
+
+        for (auto elem_it = mrModelPart.Elements().ptr_begin(); elem_it != mrModelPart.Elements().ptr_end(); ++elem_it)
+        {   
+            if ((*elem_it)->Is(STRUCTURE)){
```

Comment:
here you already use the correct one

Reply:
same, he is actually checking whether the element is trailing edge, not whether it is part of the airfoil.